packages/sources/css/src/components/forms/text-input/src/index.css

```diff
@@ -93,13 +92,12 @@ textarea.vtmn-text-input {
   color: var(--vtmn-semantic-color_content-secondary);
   font-size: rem(14px);
   line-height: 1;
-  margin-top: 0;
+  margin-block-start: rem(4px);
   width: fit-content;
 }
 
 .vtmn-text-input_helper-text--error {
   color: var(--vtmn-semantic-color_content-primary);
-  margin-top: rem(-2px);
```

Comment:
It seems this was intended to keep height consistency between classic helper text (without icon) and alert ones (with icons)

By removing that negative value you will loose that height.

https://user-images.githubusercontent.com/22882255/151188877-15e96647-6144-4d85-a44e-b7ce077a5b06.mov

That can be kept though. By removing the top value of the alert icon and adding it some vertical-align